src/api/index.js

```diff
@@ -10,7 +10,7 @@ const getAccountTransactions = require('./ledger/transactions');
 const getTrustlines = require('./ledger/trustlines');
 const getBalances = require('./ledger/balances');
 // const getPathFind = require('./ledger/pathfind');
-// const getOrders = require('./ledger/orders');
+const getOrders = require('./ledger/orders');
```

Comment:
Yeah, I copied the `api` directory from ripple-rest and now I'm going through converting all the methods to the schemas and conventions of RippleAPI.